docs/content/preview/reference/drivers/java/yugabyte-jdbc-reference.md

```diff
@@ -183,10 +183,34 @@ Create a universe with a 3-node RF-3 cluster with some fictitious geo-locations
 
 ```sh
 $ cd <path-to-yugabytedb-installation>
-
-./bin/yb-ctl create --rf 3 --placement_info "aws.us-west.us-west-2a,aws.us-west.us-west-2a,aws.us-west.us-west-2b"
 ```
 
+To create a multi-zone cluster, do the following:
+
+1. Start the first node by running the yugabyted start command, passing in the `--cloud_location` and `--fault_tolerance` flags to set the node location details, as follows:
+
+    ```sh
+    ./bin/yugabyted start --advertise_address=127.0.0.1 \
+        --base_dir=$HOME/yugabyte-2.20.0.1/node1 \
+        --cloud_location=aws.us-east-1.us-east-1a
+    ```
```

Comment:
Existing `yb-ctl` cluster had the placements as `aws.us-west.us-west-2a`, which is changed here to `aws.us-east-1.us-east-1a`
Same for other places also.

Reply:
Does that matter ? Can we not use the new ones ?

Reply:
I think in the Topology aware example they are using the earlier placements. That would error out. One workaround could be to change the placement value in that example also.

Reply:
Link to the example: https://docs.yugabyte.com/stable/reference/drivers/nodejs/yugabyte-pg-reference/#check-topology-aware-load-balancing